validity/squashfs.go

```diff
@@ -124,11 +124,11 @@ func checkSquashfsMount() error {
 	defer func() {
 		if output, err := exec.Command("umount", "-l", tmpMountDir).CombinedOutput(); err != nil {
 			// os.RemoveAll(tmpMountDir) will fail here if umount fails
-			logger.Noticef("cannot unmount sanity check squashfs image: %v", osutil.OutputErr(output, err))
+			logger.Noticef("cannot unmount validity check squashfs image: %v", osutil.OutputErr(output, err))
 		}
 	}()
 
-	// sanity check the
+	// validity check the
```

Comment:
```suggestion
	// validity check
```
It could also be 'precondition' based on the function context